k8s/namespaces/jenkins/patches/mgmt-sandbox/cluster-00/jenkins.yaml

```diff
@@ -20,7 +20,7 @@ spec:
           value: https://cftsbox-intsvc.vault.azure.net
       sidecars:
          configAutoReload:
-           image: shadwell/k8s-sidecar:0.0.7
+           image: kiwigrid/k8s-sidecar:0.1.132
```

Comment:
can't we just remove this config and go with whats in the chart?
```suggestion
```

Reply:
it's currently overridden in the main jenkins hr file